cpp/ql/src/experimental/Likely Bugs/DerefNullResult.ql

```diff
@@ -0,0 +1,31 @@
+/**
+ * @name Null dereference from a function result
+ * @description A function parameter is dereference,
+ *              while it comes from a function that may return NULL,
+ *              and is not checked for nullness by the caller.
+ * @kind problem
+ * @id cpp/deref-null-result
+ * @problem.severity recommendation
+ * @tags reliability
+ *       security
+ *       external/cwe/cwe-476
+ */
+
+import cpp
+import semmle.code.cpp.dataflow.DataFlow
```

Comment:
`semmle.code.cpp.dataflow.DataFlow` has been deprecated in favor of `semmle.code.cpp.dataflow.new.DataFlow`. Would it be possible to use that library instead? It should just be a matter of changing the import since the new library should be strictly better than the old one 🤞

```suggestion
import semmle.code.cpp.dataflow.new.DataFlow
```

Reply:
Just pushed that